chapter3.adoc

```diff
@@ -113,7 +113,8 @@ every bit position in the `SDID` field, then reading back the value in `mttp`
 to see which bit positions in the `SDID` field hold a one. The
 least-significant bits of `SDID` are implemented first: that is, if `SDIDLEN` >
 0, `SDID`[`SDIDLEN`-1:0] is writable. The maximal value of `SDIDLEN`, termed
-`SDIDMAX`, is 6 for `Smmtt[34 | 46 | 56]`.
+`SDIDMAX`, is 6 for `Smmtt[34 | 46 | 56]` and consequently allows for up to
+64 supervisor domains.
```

Comment:
My reading of the spec is that SDIDs are intended to work like ASIDs: they can be reused if you run out, at the cost of requiring a full-address-space fence when repurposing an SDID for a different supervisor domain (either at startup or during the context switch). So there is effectively no limit on the number of active supervisor domains. I have opened #76 to clarify this.

Reply:
thats correct - see the comment on the use of SDIDs in that issue #76